### PR TITLE
Lr scheduler

### PR DESCRIPTION
**Pull request by MylesBartlett**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -57,13 +58,14 @@ def parse_arguments(raw_args=None):
...
 
-    parser.add_argument('-lr', '--learning_rate', default=0.005, type=float,
+    parser.add_argument('-lr', '--learning_rate', default=0.001, type=float,
                         metavar='LR', help='initial learning rate (default: 0.005)')
 
     # model parameters
     parser.add_argument('--loss', type=str, default='MSELoss')
     parser.add_argument('--model', type=str, default='SimpleCNN')
     parser.add_argument('--optim', type=str, default='Adam')
+    parser.add_argument('--lr_step_size', type=int, default=30)
```
Comment (tmke8) [inline main.py:68]: Default should be non-surprising; so I suggest setting default to 1000 so that it normally doesn't trigger.

File: main.py [hunk 2]
```diff
@@ -136,6 +139,8 @@ def main(args):
     criterion = criterion_constructor(args) if args.args_to_loss else criterion_constructor()
     criterion = criterion.cuda() if args.cuda else criterion
 
+    scheduler = StepLR(optimizer, step_size=args.lr_step_size, gamma=0.1)
```
Comment (tmke8) [inline main.py:142]: `gamma` should be configurable with a flag.

**Review by tmke8 — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (tmke8) [reply main.py:68]: Or set `gamma` by default to 1.
